.github/workflows/wait-for-as-server-to-start.bash

```diff
@@ -7,13 +7,13 @@ set -o pipefail
 container_name=$1
 is_security_enabled=$2
 
-while true; do
-    if [[ $is_security_enabled == true ]]; then
-        # We need to pass credentials to asinfo if server requires it
-        # TODO: passing in hardcoded credentials since I can't figure out how to use --instance with global astools.conf
-        user_credentials="--user=admin --password=admin"
-    fi
+if [[ $is_security_enabled == true ]]; then
+    # We need to pass credentials to asinfo if server requires it
+    # TODO: passing in hardcoded credentials since I can't figure out how to use --instance with global astools.conf
+    user_credentials="--user=admin --password=admin"
```

Comment:
store the user and password credentials in a github secret

Reply:
This is the default credentials for the Aerospike EE server with security enabled. I don't think using secrets here is necessary unless we change those credentials